sdk-python/create_bulk_messages.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+import os
+import uuid
+
+from awaazde import AwaazDeAPI
+from awaazde.constants import CommonConstants
+from awaazde.utils import CSVUtils
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("username", help="Username", type=str)
+    parser.add_argument("password", help="Password", type=str)
+    parser.add_argument("organization", help="Organization ", type=str)
+    parser.add_argument("csv_file_path", help="Path for the csv ", type=str)
+    parser.add_argument('--use_custom_format', type=bool, default=False,
+                        help="Set this as True if the messages are of a custom Xact Template")
+    args = parser.parse_args()
+    return args
+
+
+def get_next_tag(headers):
```

Comment:
I think with introduction of `--use_custom_format`, I believe this entire logic should be removed